compiler/packages/babel-plugin-react-compiler/src/HIR/visitors.ts

```diff
@@ -851,7 +851,8 @@ export function mapTerminalSuccessors(
         loc: terminal.loc,
       };
     }
-    case "scope": {
+    case "scope":
+    case "pruned-scope": {
       const block = fn(terminal.block);
       const fallthrough = fn(terminal.fallthrough);
       return {
```

Comment:
Hi @josephsavona, would it be appropriate to change the value of the return value's `kind` to `terminal.kind`?